README.md

```diff
@@ -1,12 +1,19 @@
-# speechmatics_python
-Example script (supported) to help you integrate with our API
+# Legacy V1 Python Example
+
+Example script (supported) to help you integrate with our legacy V1 API.
 
 This code was developed and tested with Python 2.7.12 and Python 3.5.2   
 
 This is an example script for uploading audio files and saving the transcript in a .json file  
 
+> NOTE: For information on how to use the current V2 Speechmatics API,
+> see [speechmatics-python](https://github.com/speechmatics/speechmatics-python) repository.
```

Comment:
speechmatics-python does not support V2 SaaS API yet, only realtime

Reply:
Rephrased to `Speechmatics Realtime ASR API v2` (matching the description in https://github.com/speechmatics/speechmatics-python/blob/master/README.md)